app/models/medium.rb

```diff
@@ -102,6 +100,18 @@ def put_file_with_retry(file_path, opts={}, num_retries=5)
     raise e
   end
 
+  def export_medium
+    type.match?(EXPORT_MEDIUM_TYPE_REGEX)
+  end
+
+  def validate_content_type
+    errors.add(:content_type, 'Invalid file type') unless ALLOWED_CONTENT_TYPES.include?(content_type)
```

Comment:
should we supply the list of allowed mime types (or link to code) in the error, helpful error messages that include details avoid the client having to guess what the valid mime types are.